src/main/java/com/conveyal/datatools/common/utils/AWSUtils.java

```diff
@@ -41,6 +49,55 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(AWSUtils.class);
     private static final int REQUEST_TIMEOUT_MSEC = 30 * 1000;
+    public static final long DEFAULT_EXPIRING_AWS_ASSET_VALID_DURATION_MILLIS = 800 * 1000;
+
+    private static final AmazonEC2 DEFAULT_EC2_CLEINT = AmazonEC2Client.builder().build();
+    private static final AmazonElasticLoadBalancing DEFAULT_ELB_CLIENT = AmazonElasticLoadBalancingClient
+        .builder()
+        .build();
+    private static final AmazonIdentityManagement DEFAULT_IAM_CLIENT = AmazonIdentityManagementClientBuilder
+        .defaultClient();
```

Comment:
I'm a little concerned about building all the default clients statically. Since we're completely overhauling the AWS service integration, it would be great if we could address https://github.com/ibi-group/datatools-server/issues/269 while we're at it, but perhaps we should just save that for later. Ideally, I think we wouldn't initialize these clients until they're called for in the code.

Reply:
Since so much of this project is heavily dependent on AWS infrastructure, it seems like this is asking for datatools-server to work without AWS in general. I think that is out-of-scope for this PR.